tests/handler.test.js

```diff
@@ -0,0 +1,12 @@
+import * as handler from '../handler';
+
+test('hello', async () => {
```

Comment:
should that have a better test name?

Reply:
Actually this test does not make sense i removed it.